modules/configuration/ajax/process.php

```diff
@@ -77,18 +77,20 @@
         // This is different from the above is_numeric case; this makes use of
         // Config.ID, not Config.ConfigID (which is a FK to ConfigSettings.ID).
         // The Config table is the one that will be modified here.
-        $keySplit         = explode("-", $key); // e.g. 'add-10' or 'remove-49'
+        $keySplit         = explode("-", $key); // e.g. 'add-17-1' or 'remove'
```

Comment:
the example on this line doesnt seem to match the behaviour below.

I think it should be `add-17' or 'remove-1'`

Reply:
nevermind, all good